internal/index/indexoperations/index.go

```diff
@@ -18,11 +18,17 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
+	"regexp"
 
 	"github.com/pkg/errors"
 
 	"sigs.k8s.io/krew/internal/gitutil"
+	"sigs.k8s.io/krew/pkg/constants"
+)
+
+var (
+	NoIndexMessage   = "No index configured, run kubectl krew index add default " + constants.IndexURI + " to get started"
```

Comment:
Let's not do this here. We should ideally implement this in cmd/krew pkg as a `func PrintDefaultIndexSuggestion()`.

Index commands are for pro-users, so most users should be realizing they miss an index while using main user cmds (install, search, update etc). "index" cmds are mgmt cmds, so they ideally shouldn't print these.

Reply:
I'll just remove this for now then since I'm not covering the main user cmds in this PR.

Reply:
Yeah sgtm. We’ll handle the scenario later.